govc/importx/ovf.go

```diff
@@ -176,6 +191,7 @@ func (cmd *ovfx) NetworkMap(e *ovf.Envelope) ([]types.OvfNetworkMapping, error)
 		if m.Network == "" {
 			continue // Not set, let vSphere choose the default network
 		}
+		cmd.validateNetwork(e, m)
```

Comment:
So we continue code execution and only print the warning then?

Reply:
Right, since it may not cause any errors, plus an imported vm's vNICs could be reconfigured post-import before being powered on.  We do similar here with warnings from the API: https://github.com/vmware/govmomi/blob/9af23072767a6d3093c000464d015120e6c6d20c/govc/importx/ovf.go#L279